packages/ocp-index/ocp-index.1.1.1/opam

```diff
@@ -23,7 +23,7 @@ depends: [
   "re"
   "cmdliner"
 ]
-depopts: "lambda-term"
+depopts: "lambda-term" {< "2.0"}
```

Comment:
You put constraints in `depopts`, this is not what you want. You want to use the `conflicts` field instead.

Reply:
Thanks, Kate.